react/features/base/settings/middleware.ts

```diff
@@ -1,235 +0,0 @@
-/* eslint-disable lines-around-comment */
-import _ from 'lodash';
-import { AnyAction } from 'redux';
-
-import { IStore } from '../../app/types';
-import { PREJOIN_INITIALIZED } from '../../prejoin/actionTypes';
-import { setPrejoinPageVisibility } from '../../prejoin/actions';
-import { APP_WILL_MOUNT } from '../app/actionTypes';
-import { setAudioOnly } from '../audio-only/actions';
-import { SET_LOCATION_URL } from '../connection/actionTypes'; // minimize imports to avoid circular imports
-import { getJwtName } from '../jwt/functions';
-import { participantUpdated } from '../participants/actions';
-import { getLocalParticipant } from '../participants/functions';
-import MiddlewareRegistry from '../redux/MiddlewareRegistry';
-import { parseURLParams } from '../util/parseURLParams';
-
-import { SETTINGS_UPDATED } from './actionTypes';
-import { updateSettings } from './actions';
-import { handleCallIntegrationChange, handleCrashReportingChange } from './functions';
-import { ISettingsState } from './reducer';
-
-
-/**
- * The middleware of the feature base/settings. Distributes changes to the state
- * of base/settings to the states of other features computed from the state of
- * base/settings.
- *
- * @param {Store} store - The redux store.
- * @returns {Function}
- */
-MiddlewareRegistry.register(store => next => action => {
-    const result = next(action);
-
-    switch (action.type) {
-    case APP_WILL_MOUNT:
-        _initializeCallIntegration(store);
-        _initializeShowPrejoin(store);
-        break;
-    case PREJOIN_INITIALIZED: {
-        _maybeUpdateDisplayName(store);
-        break;
-    }
-    case SETTINGS_UPDATED:
-        _maybeHandleCallIntegrationChange(action);
-        _maybeSetAudioOnly(store, action);
```

Comment:
Was this intentionally removed? (I don't see it in any, native or web)

Reply:
Ops, setAudioOnly should be on the native one